java/samples/sample-code/src/main/java/com/microsoft/semantickernel/samples/Config.java

```diff
@@ -64,9 +65,11 @@ public static OpenAIAsyncClient getClient() throws IOException {
     }
 
     private static Supplier<ClientSettings<?>> getOpenAIClientFromDefaultPropertiesLocations(ClientType type) {
+        LOGGER.info("Create supplier for settings of "+type);
         return () -> {
             try {
                 for (String location : DEFAULT_PROPERTIES_LOCATIONS) {
+                    LOGGER.info("Attempting config file at "+location);
```

Comment:
First info should be inside the Supplier body - tell me when the Supplier is called, not when the getOAICFDPL method is called. I think this should be debug, not info. 
Second log should be debug, for sure. Don't annoy me with what the code is doing unless I decide that. 
The one log.info I'd have is one that says what file you are using. Maybe that is already in AIProviderSettings.get* methods.

Reply:
This code is part of the sample, not the library. The reason I went with `info` is because right now there is no `logging.properties` configuration file (or something similar). 

But this did raise a challenge with the current AISettings API design: once an instance of `ClientSettings` is available and `isValid()`, it is not possible to know where it was loaded from.

@johnoliver and @milderhc, should we reevaluate this?

Reply:
There's actually an xml already. I missed that. Changes made.